### PR TITLE
python sdk docs updates

### PR DESCRIPTION
**Pull request by martyna-mindsdb**
## Description

python sdk docs updates

**Fixes** #-

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [x] 📄 This change requires a documentation update

## Verification Process

To ensure the changes are working as expected:

 - [x]   Test Location: `/sdk_python/*`
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [ ] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [x] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

File: docs/sdk_python/create_model.mdx [hunk 1]
```diff
@@ -25,17 +25,32 @@ my_model = project.create_model (
...
       name='sentiment_classifier',
-      engine='huggingface',
+      engine='openai', # alternatively: engine=server.ml_engines.openai
       predict='sentiment',
+      prompt_template='answer this question: {{questions}}'
       options={
-          'model_name':'cardiffnlp/twitter-roberta-base-sentiment',
-          'input_column': 'comment',
-          'labels': ['negative', 'neutral', 'positive']
+          'model_name':'gpt4'
```
Comment (ea-rus) [inline docs/sdk_python/create_model.mdx:37]: it is possible to move 'model_name' also from options. All unknown params are recognized as options now. So these commands are equal:

```
sentiment_classifier = project.create_model (
      name='sentiment_classifier',
      engine='openai', # alternatively: engine=server.ml_engines.openai
      predict='sentiment',
      options={
          'prompt_template':'answer this question: {{questions}}',
          'model_name':'gpt4'
      }
)

sentiment_classifier = project.create_model (
      name='sentiment_classifier',
      engine='openai', # alternatively: engine=server.ml_engines.openai
      predict='sentiment',
      prompt_template='answer this question: {{questions}}',
      model_name = 'gpt4'
)
```

File: docs/sdk_python/create_model.mdx [hunk 2]
```diff
@@ -25,17 +25,32 @@ my_model = project.create_model (
 )
 ```
 
-Here is the syntax to create an OpenAI model:
+Please note that in the case of LLM models, the parameters are stored in `options`. Here is the syntax to create an OpenAI model:
 
 ```bash
-sentiment_classifier = project.create_model(
+sentiment_classifier = project.create_model (
```
Comment (ea-rus) [inline docs/sdk_python/create_model.mdx:31]: also the new syntax is :
`sentiment_classifier = project.models.create (`

File: docs/sdk_python/delete_file.mdx [hunk 3]
```diff
@@ -5,10 +5,13 @@ sidebarTitle: Remove a File
 
 ## Description
 
-In MindsDB, files are treated as tables. These are stored in the default `files` database. To delete a file, you must save this `files` database into a variable and then, run the `delete_table()` function on it.
+In MindsDB, files are treated as tables. These are stored in the default `files` database. To delete a file, you must save this `files` database into a variable and then, run the `drop_table()` function on it.
```
Comment (ea-rus) [inline docs/sdk_python/delete_file.mdx:8]: drop is a new feature and I didn't implement function for old syntax.
It means only this option is available:
`files_db.tables.drop(name)`
And no `drop_table` function exists

File: docs/sdk_python/delete_table.mdx [hunk 4]
```diff
@@ -3,6 +3,18 @@ title: Remove a Table
...
+## Syntax
+
+Here is the syntax:
+
+```sql
+# option 1
+data_source.drop_table('table_name')
+
+# option 2
+my_table.delete()
```
Comment (ea-rus) [inline docs/sdk_python/delete_table.mdx:19]: delete and drop are not the same:
delete - is deleting rows in table. 
`db.tables.drop('table2')`
drop - is dropping the whole table
`db.tables.table2.delete(a=1, b=2)`

File: docs/sdk_python/list_ml_handlers.mdx [hunk 5]
```diff
@@ -7,6 +7,6 @@ Here is how you can fetch all available ML handlers directly from Python code:
 
 ```
 mindsdb = server.get_project('mindsdb')
-data_handlers = mindsdb.query('SHOW HANDLERS WHERE type = \'ml\'')
-print(data_handlers.fetch)
+ml_handlers = mindsdb.query('SHOW HANDLERS WHERE type = \'ml\'')
```
Comment (ea-rus) [inline docs/sdk_python/list_ml_handlers.mdx:10]: I can create collection for ml handlers (similar to ml_engines):
`con.ml_handlers.list()
`
Shall I do this?

File: docs/sdk_python/list_ml_handlers.mdx [hunk 6]
```diff
@@ -7,6 +7,6 @@ Here is how you can fetch all available ML handlers directly from Python code:
...
+print(ml_handlers.fetch)
```
Comment (ea-rus) [inline docs/sdk_python/list_ml_handlers.mdx:11]: fetch is a function, it has to be called:
`print(ml_handlers.fetch())`

File: docs/sdk_python/create_model.mdx (hunk 2 above)
Comment (martyna-mindsdb) [reply docs/sdk_python/create_model.mdx:31]: So it is also `project.models.get()` instead of `get_model()`.

**Review by ea-rus — APPROVED** [review]

**Pull request merged**